app-front-ui-goj/src/app/pages/Map.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import { MapContainer, TileLayer, Marker, Popup, useMap } from "react-leaflet";
+import osm from "./osm-provides";
+import "leaflet/dist/leaflet.css";
+import L from "leaflet";
+import useGeoLocation from "../hooks/useGeoLocation";
+
+const markerIcon = new L.Icon({
```

Comment:
Create in assets separate file "Assests.tsx" and add there this const as export and only import it